### PR TITLE
Docs for Adyen

### PR DESCRIPTION
**Pull request by Fifciu**
### Related Issues

closes #EN-132

### CODE REVIEW

File: packages/core/docs/integrations/README.md [hunk 1]
```diff
@@ -120,10 +120,9 @@ Click on the integration tile to read its documentation.
     name="Adyen"
     image="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a2/Adyen_Corporate_Logo.svg/1280px-Adyen_Corporate_Logo.svg.png"
     category="Payment"
-    isOpenSource="true"
-    isWip="true"
-    :from-core="true"
```
Comment (filrak) [inline packages/core/docs/integrations/README.md:125]: why you removed from-core ?

File: packages/core/docs/integrations/adyen.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
+# Adyen <Badge text="Enterprise" type="info" />
+
+::: warning Paid feature
+This feature is part of the Enterprise version. Please [contact our team](https://www.vuestorefront.io/contact/sales) if you'd like to use it in your project.
+:::
```
Comment (filrak) [inline packages/core/docs/integrations/adyen.md:5]:
```suggestion
>This feature is part of the Enterprise version. Please [contact our team](https://www.vuestorefront.io/contact/sales) if you'd like to use it in your project.
```
for consistency with other docs

File: packages/core/docs/integrations/adyen.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+```ts
+type PaymentAndOrder = Payment & { order: Order }
+```
+
+5. Add `origin` to allowed origins in Adyen's dashboard. You can do it in the same place where you looked for the `clientKey`.
+
+6. Commercetools shares [Adyen integration](https://github.com/commercetools/commercetools-adyen-integration) which should be deployed as an independent app. We recommend to deploy it as a Google Function or a AWS Lambda.
+
+::: warning
```
Comment (filrak) [inline packages/core/docs/integrations/adyen.md:123]: why this is a warning?

Comment (Fifciu) [reply packages/core/docs/integrations/adyen.md:125]: Wanted to emphasize it this way

**Review by filrak — APPROVED** [review]

**Pull request merged**